Sources/Controllers/2.0/RoutePlanning/InitialRoutePlanningBottomSheetViewController.mm

```diff
@@ -69,7 +71,17 @@ - (void) applyLocalization
 
 - (CGFloat) initialHeight
 {
-    return DeviceScreenHeight - DeviceScreenHeight / ([OAGPXDatabase sharedDb].gpxList.count > 1 ? 3 : 2);
+    int tracksCount = (int)[OAGPXDatabase sharedDb].gpxList.count;
+    int maxHeight = DeviceScreenHeight / 3 * 2;
+    
+    int estimatedHeight = 60 + 18 + 2 * 48 + 16 + 60 + OAUtilities.getBottomMargin;
```

Comment:
Please never hardcode the values like this. This is wrong for multiple reasons: it is terrible for readability, it does not take multi-line cells into account. It is a very bad approach and is likely to cause strange behavior on devices with smaller screens. A solution would be to add an approximate constant value + constant value for each gpx but not the magic numbers!